BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -104,7 +103,7 @@ private void updateLocation() {
     }
 
     private void onLocationUpdated(Location location) {
-        if (maximumMovementSpeed > 0 && lastKnownLocation != null) {
+        if (maximumMovementSpeed > _NOT_SET && lastKnownLocation != null) {
```

Comment:
Use `!=` instead of `>`. It's not wrong with the current implementation, but you assume that the value is always negative. If someone refactors this and assigns `MAXIMUM_MOVEMENT_SPEED_NOT_SET  = Double.MAX_VALUE`, this would break.